debsources/app/templates/news_archive.html

```diff
@@ -14,24 +14,41 @@
 
 {% block content %}
 
-<div id="indexmenu">
 
-  <h1 class="h1mainpage">{{ self.title() }}</h1>
-
-  <div class="content" style="text-align: left">
+<div class="row">
+  <div class="col-md-12">
+    <h1 class="h1mainpage">{{ self.title() }}</h1>
     {% if archived_news %}
       {{ archived_news|safe }}
     {% else %}
-      <h3>Looks like the news archive is empty!</h3>
+    <h3>Looks like the news archive is empty!</h3>
     {% endif %}
   </div>
 </div>
+
 {% endblock %}
 
-{# show the background debian image #}
+{# XXX: This is the old way bg_wrapping is handled. The new modified way uses
+        super() to change the image.
+        If this is rejected this block of code will be needed.
+
+show the background debian image
```

Comment:
i don't think we need comments like this in the templates. i you want you can add these issues on trello so we can see them as actionable items etc.